Bottom: 61f020e440f87bd2d1a531d27778b6ac5e776b0c
Top:    f9683c3806647157fc88f75f0c73c5d676d93f79
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:58:55 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
